docs/guides/help-for-the-non-technical.md

```diff
@@ -0,0 +1,196 @@
+# Help for our less-technical users
+
+Here at Redocly, we want everyone to feel comfortable using our products. If you're new to open source, command lines, GitHub and docs-as-code, then this guide is for you!
+
+:::success Tip
+We assume that you already have a basic understanding of what an API is.
+:::
+
+We recommend reading this guide before you start installing and using our open-source products, so you have the confidence to get the most out of what are (in our humble opinions) some pretty cool tools.
+
+Now, let's get into it!
+
+## What is open-source software?
+
+[opensource.com](https://opensource.com/) nails this definition in a single sentence:
+
+> Open source software is software with source code that anyone can inspect, modify, and enhance.
+
+In recent years, a huge worldwide open-source community has grown — one that values sharing, collaboration and learning. Even if you're not digging around in the codebase or contributing to the docs, by using Redocly's open-source products, you are part of the community.
+
+Redocly offers both open-source and premium products to our customers, but we started out as purely open source!
+
+## The need for well-documented APIs
+
+The growth of the internet has led to more organizations developing their own APIs in order to share data and utilize functionality developed by others. When it comes to documenting APIs, traditional approaches fall short. There are several reasons for this, but the main reason is that docs are kept apart from the code.
+
+Because APIs have no front end, the docs *are* the UI. Developers rely on up-to-date documentation published in an easy-to-read format so they can understand how an API works, and how to implement it into their code. Simply put, without docs, developers can't use an API.
+
+### Types of API documentation
+
+API docs consist of two types of content: *conceptual docs* and *reference docs*.
+
+##### Conceptual docs
```

Comment:
question: I wonder if this is all conceptual here or rather is contextual docs? It seems to cover more categories of docs like tutorials and guides.

Reply:
Tom Johnson calls them 'conceptual docs' so I just adopted his term. Would you like me to change to 'contextual docs', or maybe even 'supporting docs'? https://idratherbewriting.com/learnapidoc/docconceptual.html

Reply:
We use the term contextual docs throughout which is intentionally more vague. I don't like the conceptual term particularly because I feel it is specific to the bottom-left quadrant here: https://diataxis.fr/ and missing things like practical how-to or learning-based tutorials both of which are important. Also, some of the conceptual docs are actually already reference docs and part of those such as authentication and authorization (it's part of the APIs security definitions).

Reply:
Makes sense.